sulfnucrate.F90

```diff
@@ -4,51 +4,386 @@
 
 !!  Calculates particle production rates due to nucleation <rhompe>:
 !!  binary homogeneous nucleation of sulfuric acid and water only
-!!  Numerical method follows Zhao & Turco, JAS, V.26, No.5, 1995.
+!!  Numerical method follows one of the following:
```

Comment:
I think it would be better to break this up into 3 files following the pattern of vaporp, where there is a top level routine (vaporp.F90) that has the switch statement to select the appropriate routine, and then each routine is in its own file. So I would suggest sulfnucrate_zhao1995.F90 and sulfnucrate_vehkamaki2002.F90 as the new files. This keeps the files simpler as we are likely to add more of these in the future.